Src/Plugins/Security/SecurePlaintextSecretsValidators/SEC101_008.AwsCredentialsValidator.cs

```diff
@@ -10,7 +10,7 @@
 using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Sdk;
 using Microsoft.RE2.Managed;
 
-namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.SecurePlaintextSecretsValidators
```

Comment:
>SecurePlaintextSecretsValidators [](http://example.com/codeflow?start=71&length=32)

This namespace change isn't really serving us well (except to keep namespaces in line with directories).

Reply:
ok!
just updated.